hack/make-rules/test-e2e-node.sh

```diff
@@ -177,7 +177,7 @@ else
 
   # Test using the host the script was run on
   # Provided for backwards compatibility
-  go run test/e2e_node/runner/local/run_local.go \
+  go run --mod=vendor test/e2e_node/runner/local/run_local.go \
```

Comment:
/hold

we are not currently building in module mode (and will likely explicitly disable module mode for builds when we first move to go1.13, see proposed plan https://github.com/kubernetes/kubernetes/issues/82531)